pkg/contracts/ton/gateway.go

```diff
@@ -256,3 +115,30 @@ func parseInternalMessageBody(tx ton.Transaction) (*boc.Cell, error) {
 
 	return &body, nil
 }
+
+// GetTxFee returns maximum transaction fee for the given operation.
+// Real fee may be lower.
+func (gw *Gateway) GetTxFee(ctx context.Context, client MethodRunner, op Op) (math.Uint, error) {
+	const (
+		method  = "calculate_gas_fee"
+		symType = "VmStkTinyInt"
```

Comment:
Can we add a comment to explain what does it represent?

Reply:
typo: `symType` -> `sumType`